### ISSUE TITLE
Output from 1.18 snapshots (after at least experimental 7) is blank

### ISSUE THREAD

**Issue by rjp** [spoutn1k/mcmap#71]
Minecraft Overviewer has the same issue (https://github.com/overviewer/Minecraft-Overviewer/issues/1982) which suggests there's been some internal NBT shenanigans since 1.18 experimental-snapshot-7 (works fine).

**Comment by spoutn1k** [comment]
Yep ! Its being addressed in the ![compatibility](https://github.com/spoutn1k/mcmap/tree/compatibility) branch. Mojang made some changes to accomodate the 3D biomes.

EDIT: Mistakenly closed the issue. I'll close it when the branch is merged into master.

Also for future reference, if anyone ever wants to check the shenanigans for themselves, you can easily do so with the scripts provided in this repo:

```
./bin/extractChunk ~/.minecraft/saves/your_save/region/r.0.0.mca 0 0 | ./bin/nbt2json
```

If run into the build directory, this should output a whole chunk into JSON format. You can select the chunk by changing the region/the chunk coordinates in the region.

**Comment by spoutn1k** [comment]
Merged with #73 ! Please configure with `cmake -DSNAPSHOT=1` to enable the snapshots. I made it so that it is easier to add new version support, so it should be faster from now on.
